src/components/user-card/UserCard.js

```diff
@@ -15,3 +15,20 @@ import React from 'react';
 //  этот div будет содержать:
 //  - блок h4 с классом card-title, он должен вывести имя и фамилию (first_name, last_name)
 //  - блок div c классом "card-text", содержащий 2 div блока для поля email и поля address
+export const UserCard = (props) => {
+    const {user} = props;
+    if (!user) return null;
+    const {first_name, last_name, email, address, _links} = user;
+    return (
+        <div className={'may-user-card card'}>
+            <img src={_links.avatar.href} className={'may-user-card-avatar rounded-circle'}/>
+            <div className={'card-body'}>
```

Comment:
if you want to pass some string as a prop you can not to use curly brackets {}
and it would be `className='card-body`